cfme/services/catalogs/catalog_item.py

```diff
@@ -112,23 +197,11 @@ def __init__(self, item_type=None, vm_name=None, name=None, description=None,
         self.provider_type = provider_type
         self.provisioning_data = prov_data
         self.domain = domain
-        Navigatable.__init__(self, appliance=appliance)
-
-    def __str__(self):
-        return self.name
 
     def create(self):
-        # Create has sequential forms, the first is only the provider type
-        navigate_to(self, 'Add')
-        # For element not found exception (To be removed)
-        sel.sleep(5)
+        view = navigate_to(self, 'Add')
         sel.select("//select[@id='st_prov_type']",
                    self.provider_type or self.item_type or 'Generic')
```

Comment:
This field comes after adding and before the form . 
View.fill(thisfield) does not work.